README.md

```diff
@@ -32,7 +44,7 @@ log4j.appender.A1.layout.ConversionPattern=%d [%t] %-5p %c - %m%n
 #log4j.logger.edu.umich=INFO
 log4j.logger.edu.umich=DEBUG 
 ```
-7.  For Adding Build information to the Project Release candidate populate src/main/webapps/build.txt with information about the current build (GIT commit, build time, etc.).
+8. For Adding Build information to the Project Release candidate populate src/main/webapps/build.txt with information about the current build (GIT commit, build time, etc.).
     If using Jenkins to build the application, the following script placed in the "Execute Shell" part of the "Pre Steps" section would do the job:
```

Comment:
Why not just add it to the build script?  One way or another it can run locally too.